packages/core/src/lib/test/is-empty-record.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it, expect } from 'vitest';
+import { isEmptyRecord } from '../util/is-empty-record';
+
+describe('isEmptyRecord', () => {
+  it('should return false if value is null', () => {
+    expect(isEmptyRecord(null)).toBe(false);
+  });
+  it('should return false if value is undefined', () => {
+    expect(isEmptyRecord(undefined)).toBe(false);
+  });
+
```

Comment:
if you use new line to create groups of tests, you can also use nested describes. In that way you can give your group also a name.

Reply:
That was a co-incidence sorry. I did not intentionally wanted to introduce grouping